docs/examples/.env.truenas.example

```
@@ -0,0 +1,64 @@
+
+# ============================================
+# SECRETS (CRITICAL - generate new values!)
+# ============================================
+# Generate with: openssl rand -hex 32
+SECRET_KEY=CHANGE-ME-USE-openssl-rand-hex-32
+
+# ============================================
+# DATABASE
+# ============================================
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)